openmc_plotter/plotmodel.py

```diff
@@ -116,10 +118,39 @@ def __init__(self):
         # reset random number seed for consistent
         # coloring when reloading a model
         reset_seed()
-
         self.previousViews = []
         self.subsequentViews = []
-        self.defaultView = self.getDefaultView()
+
+        if use_settings_pkl and os.path.isfile('plot_settings.pkl'):
+            with open('plot_settings.pkl', 'rb') as file:
+                model = pickle.load(file)
+
+                # check GUI version
+                if model.version != self.version:
+                    print("WARNING: previous plot settings are for a different "
+                        "version of the GUI. They will be ignored.")
+                    wrn_msg = "Existing version: {}, Current GUI version: {}"
+                    print(wrn_msg.format(model.version, self.version))
+                    view = None
+                else:
+                    view = model.currentView
+
+                    # restore statepoint file
+                    try:
+                        self.statepoint = model.statepoint
```

Comment:
Since all we're using is `version`, `currentView`, and `statepoint`, it doesn't make sense for us to pickle the entire `PlotModel` object. Instead, we could just pickle a dictionary with the attributes we need. Something like:
```Python
pickle_data = {
    'version': self.model.version,
    'currentView': self.model.currentView,
    'statepoint': self.model.statepoint
}
pickle.dump(pickle_data, file)
```
This should cut down substantially on the size of the pickle files.

Reply:
Yes- As long as we don't care about the `previousViews` and `subsequentViews`, then this can be shrunk a lot.

Reply:
Personally I think it's reasonable to not save those. I don't think I've ever relied on the ability to go to a previous view after reloading.